website/content/api-docs/system/mounts.mdx

```diff
@@ -156,9 +156,11 @@ This endpoint enables a new secrets engine at the given path.
   - `audit_non_hmac_response_keys` `(array: [])` - List of keys that will not be
     HMAC'd by audit devices in the response data object.
 
-  - `listing_visibility` `(string: "")` - Specifies whether to show this mount
-    in the UI-specific listing endpoint. Valid values are `"unauth"` or
-    `"hidden"`. If not set, behaves like `"hidden"`.
+  - `listing_visibility` `(string: "")` - Specifies the information to show about
+   this mount in the unauthenticated UI-specific listing endpoint. Valid values are:
+      - `"hidden"`: hides mount data from list, equivalent to the default `""` 
+      - `"unauth"`: returns mount path, description and method type
+      - `"dropdown"`: returns method type (used to customize login form dropdown in the UI)
```

Comment:
Yes, that's the intended use-case. I was a little confused here. Both auth methods and mounts are "mounted" so `listing_visibility` is a parameter that can be sent when configuring either?

I updated the documentation here for consistency, though I can probably remove it since it's not applicable.

Reply:
Or a note that the value is accepted but does nothing in the mount case.

Reply:
Okay! Sounds good ☺️

Reply:
@ncabatoff I've pushed a commit removing `dropdown` from auth mounts, LMKWYT. :-)